docs/commands/rhoas_service-registry_role_add.md

```diff
@@ -22,7 +28,7 @@ rhoas service-registry role add --role=admin --username=joedough
 
 ```
       --instance-id string       ID of the Service Registry instance to be used (by default, uses the currently selected instance)
-      --role string              Role to apply
+      --role admin               Role to apply: admin, `manager`, or `viewer`
```

Comment:
That looks incorrect - `string` is supposed to indicate the data type, but it has been changed to admin.

```suggestion
      --role string               Role to apply: admin, `manager`, or `viewer`
```

Reply:
this will also break ascidocs lets not use /`

Reply:
Seems that the backticks caused the change from string to admin -- fixed now.